ssm-diff.py

```diff
@@ -3,13 +3,13 @@
 import argparse
 
 
-def init(filename):
+def init(filename, paths=['/']):
```

Comment:
Can we rely on a default=['/'] option in a parser to avoid having this in multiple places?